doc/specs/staffPage.md

```diff
@@ -0,0 +1,90 @@
+# Staff Page
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+4. Data model
+
+## Motivation
+
+>The staff team is the core of the smooth operation of the site. The staff page is a list of the site's staff members for the user's convenience.
+
+## Business Rules
+
+> * User classes must have an option to be presented on the staff page.
```

Comment:
You mention this once but never again in the use cases. Be more verbose about the staff page, that's not descriptive, it has fls members but thats also summarized in this one business rule. All the use cases can use improvement.